agignore

```diff
@@ -1,3 +1,4 @@
 log
 tags
 tmp
+/.git/
```

Comment:
I don't think we should ignore `.gitignore` files - I may want to find those with ctrl-p. Plus, `.gitignore` files are not in `.git` directories, and so are correctly not ignored by this line.